src/components/molecules/Common/AssetModal/AssetContainer/index.tsx

```diff
@@ -23,22 +25,26 @@ export type Props = {
   assets?: Asset[];
   isMultipleSelectable?: boolean;
   accept?: string;
-  onCreateAsset?: (file: File) => void;
+  onCreateAsset?: (files: FileList) => void;
+  onRemove?: (assets: AssetType[]) => void;
   initialAsset?: Asset;
   selectedAssets?: Asset[];
   selectAsset?: (assets: Asset[]) => void;
   fileType?: "image" | "video" | "file";
+  isSettingPage?: boolean;
 };
 
 const AssetContainer: React.FC<Props> = ({
   assets,
   isMultipleSelectable = false,
   accept,
   onCreateAsset,
+  onRemove,
   initialAsset,
   selectedAssets,
   selectAsset,
   fileType,
+  isSettingPage,
```

Comment:
```suggestion
  isHeightFxied,
```

isSettingPage is fine, but it is not versatile. If you use a name like this, it will be convenient when you want to use this property outside of SettingPage.

Reply:
ok, thanks